data/mods/Aftershock/items/armor/modular_exos/modular_exoskeleton.json

```
@@ -0,0 +1,241 @@
+[
+  {
+    "id": "afs_exo_undersuit",
+    "repairs_like": "thermal_suit",
+    "type": "ARMOR",
+    "copy-from": "jumpsuit",
+    "looks_like": "robofac_enviro_suit",
+    "name": { "str": "exoskeleton undersuit" },
+    "description": "A form-fitting suit inlaid with meant to be worn beneath power armor.  Although not a requirement for the operation of power armor, it features climate control and haptic feedback systems that offer improved performance and comfort.  The suit is not designed for use outside of armor, it has no pockets, offers nearly no protection and damages easily.",
+    "weight": "610 g",
+    "volume": "3500 ml",
+    "color": "brown",
+    "material": [ "nomex", "plastic" ],
+    "armor": [
+      { "covers": [ "head" ], "coverage": 100, "encumbrance": 5 },
+      { "covers": [ "torso" ], "coverage": 100, "encumbrance": 5 },
+      { "covers": [ "leg_l", "leg_r" ], "coverage": 100, "encumbrance": 5 },
+      { "covers": [ "arm_l", "arm_r" ], "coverage": 100, "encumbrance": 5 },
+      { "covers": [ "hand_l", "hand_r" ], "coverage": 100, "encumbrance": 5 },
+      { "covers": [ "foot_l", "foot_r" ], "coverage": 100, "encumbrance": 5 }
+    ],
+    "warmth": 10,
+    "flags": [ "USE_UPS", "SKINTIGHT", "WATERPROOF", "POWERARMOR_COMPATIBLE" ]
+  },
+  {
+    "id": "modular_exoskeleton_off",
```

```suggestion
    "id": "modular_exoskeleton",
```
The old power armor code doesnt like the _off suffix. It would probably be better to fix the old code, but as you mentioned not feelingvery comfortable with C++ I guess you could do this instead.